app/waf/package.json

```diff
@@ -23,7 +23,7 @@
     "tslint": "lb-tslint",
     "tslint:fix": "npm run tslint -- --fix",
     "pretest": "npm run clean && npm run build",
-    "test": "lb-mocha --allow-console-logs \"dist/test\"",
+    "test": "npm run prettier:fix && lb-mocha --allow-console-logs \"dist/test\"",
```

Comment:
Personally I disagree to make format fixing action to be transparent. It is not a usual practice.

Reply:
unusual reality need unusual solution. lb4's own code doesn't comply with its own formatter. 

I prefer to make the process automatically as possible. I don't want such thing to bother me every time.

Reply:
Since automated tools like Travis will run npm test, we need to prevent any possibility of undetectable code changing. If we need an automated code change is CI process, that code change must be specifically done by another set of code made by the code owner.  The output of formatting tool is not predictable, so I say it not a usual practice technically.

Reply:
Agree. OK for that.